### ISSUE TITLE
404 on /api/v1/deployments/ but not on info

### ISSUE THREAD

**Issue by JoshFalken** [magneticio/vamp#270]
Hi,

When I run vamp and execute

curl -v -X GET http://192.168.1.53:8083/api/v1/info

I get the correct JSON-message but when I try to add the blueprint via

curl -v -X POST --data-binary @sava_1_0.yaml -H "Content-Type: application/x-yaml" http://192.168.1.53:8083/api/v1/deployments/

I get the following:

* Hostname was NOT found in DNS cache
*   Trying 192.168.1.53...
* Connected to 192.168.1.53 (192.168.1.53) port 8083 (#0)
> POST /api/v1/deployments/ HTTP/1.1
> User-Agent: curl/7.35.0
> Host: 192.168.1.53:8083
> Accept: */*
> Content-Type: application/x-yaml
> Content-Length: 284
> 
* upload completely sent off: 284 out of 284 bytes
< HTTP/1.1 404 Not Found
< Date: Mon, 27 Jul 2015 17:05:14 GMT
< Content-Type: text/plain; charset=UTF-8
< Content-Length: 42
< 
* Connection #0 to host 192.168.1.53 left intact

I also get this on other endpoints I try to reach (but not on api/v1/info) It's probably something really simple but for the likes of me, I can't figure it out. Where did I go wrong?

**Comment by tnolet** [comment]
Hi @JoshFalken 

Your curl command looks fine. Ere you sure you're posting to Vamp Core? We normally run that on port 8080 or 8081. It looks like you're posting to Vamp Pulse, which by default runs on port 8083. Please post the full output of the /api/v1/info endpoint in this issue and we can clear this up.

From next week on we package the UI with Vamp Core and you don't have to use curl anymore for most operations.

**Comment by JoshFalken** [comment]
Hi @tnolet,

Thanks again for your help. I did make a mistake in thinking that I was querying the Vamp Core on port 8083. After some digging I found out that there was something wrong the certificates and that was the reason Vamp Core didn't start. So, I could have done all the requests I wanted on port 8080 :) The main logging didn't show that error as far as I know, they just stated that Vamp Core was started. But when I executed 

```
java -Dconfig.file=/vamp-core.conf -Dlogback.configurationFile=/logback.xml -jar vamp-core.jar
``` 
within the container, I did see vamp was looking voor key.pem and cert.pem. After generating  keys with these names Vamp Core started. So this problem is solved. Maybe it's an idea to run a test with missing key.pem/cert.pem and check whether or not your configuration also breaks and  doesn't give a clear error message that vamp core isn't available.
